recipes/pymatreader/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pymatreader" %}
+{% set version = "0.0.26" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 93aa09a5265fe430e43f6b085f9bd9131384c441c6600d2f9a60ce994cfee392
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - future
+    - h5py
+    - numpy
+    - python
+    - scipy !=1.7.0
+    - xmltodict
+
+test:
+  imports:
+    - pymatreader
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://gitlab.com/obob/pymatreader
+  summary: Convenient reader for Matlab mat files
+  license: BSD-2-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - larsoner
+    - thht
```

Comment:
@thht can you:

>  [post] a comment confirming [you] are willing to be listed there.

? Or if you don't want to co-maintain the recipe I can remove you. Usually there is little to nothing to do -- once something is pushed to PyPi, a bot notices, opens a PR to update the recipe, auto-merges, and auto-rebuilds. In my experience you only really have to change things when dependencies change